components/pricing_modal/content.scss

```diff
@@ -4,3 +4,32 @@
     flex-direction: column;
     justify-content: center;
 }
+
+div.flexcontainer {
+    display: flex;
+    min-height: 120px;
+}
```

Comment:
do we have to work around the css selector precedence here? instead of `.flexcontainer`

Reply:
Yeah, `.flexcontainer` can work instead of `div.flexcontainer`. I think I just saw something like that in some other areas of the codebase and I thought it was the norm (but maybe those were for repeated classnames). I'll fix that :)